scripts/upsert/sections.ts

```diff
@@ -0,0 +1,53 @@
+import { Term } from '../../lib/term';
+import { differenceInMinutes } from 'date-fns';
+import { upsertSections } from '../../lib/sections';
+import { findLatestTask, createTask } from '../../lib/task';
+import { Fetch } from '../../lib/banner/fetch';
+import makeFetchCookie from 'fetch-cookie';
+import { getSearchResults } from '../../lib/banner';
+import { range } from '../../lib/fn';
+import fetch from 'node-fetch';
```

Comment:
Fetch is actually globally patched in for scripts so it makes Next.js runtime

Reply:
<img width="675" alt="image" src="https://user-images.githubusercontent.com/50898635/197094299-36a45193-feaa-4cc8-8cae-1d8391c8a350.png">
I get this error if I remove that :/